SparkPost/support-docs
# WE-288: sitemap.xml and robots.txt automation

### What Changed
I added the `next-sitemap` plugin, a config file for the plugin, a `postbuild` script to always run the `next-sitemap` plugin after a build and updated the `.gitignore` file to ignore `sitemap.xml`, any sitemaps that are split into multiples and the `robots.txt` file.

### How to test
- run `npm run build` and verify that the `sitemap.xml` and `robots.txt` files are built and with the appropriate parameters
    - `robots.txt` discourages web crawlers (like Spiderman and the like. jk :-P )
    - `sitemap.xml` has around 3k+ files all in one file (we have a `sitemapSize` setting of 5k, so that shouldn't break into multiple sitemaps until we hit that number)
